cmd/tsbs_load_siridb/main.go

```diff
@@ -0,0 +1,81 @@
+package main
+
+import (
+	"bufio"
+	"flag"
+	"log"
+
+	"github.com/timescale/tsbs/load"
+)
+
+const (
+	account       = "sa"
```

Comment:
since these constants are used in `creator.go`, they should probably live there

Reply:
Yes I agree, I moved them